www/wp-config.php

```diff
@@ -0,0 +1,83 @@
+<?php
+// ===================================================
+// Load database info and local development parameters
+// ===================================================
+if ( file_exists( dirname( __FILE__ ) . '/local-config.php' ) ) {
+	define( 'WP_LOCAL_DEV', true );
+	include( dirname( __FILE__ ) . '/local-config.php' );
+} else {
+	define( 'WP_LOCAL_DEV', false );
+	define( 'DB_NAME', '%%DB_NAME%%' );
+	define( 'DB_USER', '%%DB_USER%%' );
+	define( 'DB_PASSWORD', '%%DB_PASSWORD%%' );
+	define( 'DB_HOST', '%%DB_HOST%%' ); // Probably 'localhost'
+}
+
+// ========================
+// Custom Content Directory
+// ========================
+define( 'WP_CONTENT_DIR', dirname( __FILE__ ) . '/content' );
+define( 'WP_CONTENT_URL', 'http://' . $_SERVER['HTTP_HOST'] . '/content' );
+
+// ================================================
+// You almost certainly do not want to change these
+// ================================================
+define( 'DB_CHARSET', 'utf8' );
+define( 'DB_COLLATE', '' );
+
+// ==============================================================
+// Salts, for security
+// Grab these from: https://api.wordpress.org/secret-key/1.1/salt
+// ==============================================================
+define( 'AUTH_KEY',         'put your unique phrase here' );
+define( 'SECURE_AUTH_KEY',  'put your unique phrase here' );
+define( 'LOGGED_IN_KEY',    'put your unique phrase here' );
+define( 'NONCE_KEY',        'put your unique phrase here' );
+define( 'AUTH_SALT',        'put your unique phrase here' );
+define( 'SECURE_AUTH_SALT', 'put your unique phrase here' );
+define( 'LOGGED_IN_SALT',   'put your unique phrase here' );
+define( 'NONCE_SALT',       'put your unique phrase here' );
+
+// ==============================================================
+// Table prefix
+// Change this if you have multiple installs in the same database
+// ==============================================================
+$table_prefix  = 'wp_';
+
+// ================================
+// Language
+// Leave blank for American English
+// ================================
+define( 'WPLANG', '' );
+
+// ===========
+// Hide errors
+// ===========
+ini_set( 'display_errors', 0 );
+define( 'WP_DEBUG_DISPLAY', false );
+
+// =================================================================
+// Debug mode
+// Debugging? Enable these. Can also enable them in local-config.php
+// =================================================================
+// define( 'SAVEQUERIES', true );
+// define( 'WP_DEBUG', true );
+
+// ======================================
+// Load a Memcached config if we have one
+// ======================================
+if ( file_exists( dirname( __FILE__ ) . '/memcached.php' ) )
```

Comment:
Actually, I decided having these in a separate file would be smarter. See https://github.com/misterbisson/triton-wordpress/pull/1/files#diff-161f5c73d02a10b635e264e618963653